runtime/compiler/compile/J9SymbolReferenceTable.cpp

```diff
@@ -533,6 +557,23 @@ J9::SymbolReferenceTable::findOrCreateMethodTypeTableEntrySymbol(TR::ResolvedMet
    return symRef;
    }
 
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+TR::SymbolReference *
+J9::SymbolReferenceTable::refineInvokeCacheElementSymRefWithKnownObjectIndex(TR::ResolvedMethodSymbol * owningMethodSymbol,  TR::SymbolReference * originalSymRef, uintptr_t arrayElementRef)
+   {
```

Comment:
Since we're taking an object ref, VM access should have been acquired by the caller. We need an assert here to ensure that